recipes/wand/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Wand" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "28e0454c9d16d69c5d5034918d96320d8f9f1377b4fdaf4944eec2f938c74704" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest >=2.3.0
```

Comment:
Many of the listed `run` dependencies actually look like test dependencies:

https://github.com/dahlia/wand/blob/master/setup.py#L48-L52

Reply:
Thanks. I have removed those dependencies.